### DISCUSSION TITLE
Maximum portal size

### DISCUSSION THREAD

**Discussion by Obi-Stan-Kenobi** [iPortalTeam/ImmersivePortalsMod#1366] [Q&A]
I was wondering if there is a way to increase the maximum portal size.
I believe it is 22x22 at the moment.

**Comment by qouteall** [comment]
In the latest version in Mc 1.19.4, the default maximum size is 64. In older versions it's about 20.

**Reply by Obi-Stan-Kenobi** [reply]
I am in a 1.19.2 modpack at the moment.
Is there a way for me to change the default maximum size in this version?

**Reply by qouteall** [reply]
No in 1.19.2. It's recommended to use 1.19.4 now
